api/server/router/network/network_routes.go

```diff
@@ -80,7 +80,7 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 		return err
 	}
 
-	if _, err := n.clusterProvider.GetNetwork(create.Name); err == nil {
+	if _, err := n.clusterProvider.GetNetworkByName(create.Name); err == nil {
 		return libnetwork.NetworkNameError(create.Name)
```

Comment:
Yes, that makes sense, then we can confirm if it should be fixed / changed on that PR